tools/packaging/kernel/configs/fragments/x86_64/sev/sev.conf

```diff
@@ -4,8 +4,7 @@ CONFIG_AMD_MEM_ENCRYPT=y
 CONFIG_EFI=y
 CONFIG_EFI_SECRET=m
 CONFIG_EFI_STUB=y
-CONFIG_MODULE_SIG=y
 CONFIG_MODULES=y
 CONFIG_MODULE_UNLOAD=y
 CONFIG_SEV_GUEST=y
-CONFIG_VIRT_DRIVERS=y
\ No newline at end of file
+CONFIG_VIRT_DRIVERS=y
```

Comment:
As CONFIG_MNODULE_SIG has been removed, you'll need to bump https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kernel/kata_config_version